TeensyDmx.cpp

```diff
@@ -136,6 +137,9 @@ TeensyDmx::TeensyDmx(HardwareSerial& uart, RdmInit* rdm) :
     m_rdmChecksum(0),
     m_deviceLabel{0}
 {
+    Serial.begin(115200);
```

Comment:
The library should not have logging in it, this is for the user to decide.

Reply:
Oops sorry, some of the aforementioned tidying. Is your preference to merge what I've got into master then, in which case I'll strip it, otherwise if you want to go down the separate branch route, I might try and do something with macros for now, so I don't have to keep adding and removing print statements while testing stuff.

Reply:
Done.